packages/amplication-data-service-generator/src/server/static/src/interceptors/filterResult.interceptor.ts

```diff
@@ -0,0 +1,42 @@
+import {
```

Comment:
I suggest that we call this interceptor "FilterResponse" and the other one that we are working on "ValidateRequest"